shell/edit/workload/index.vue

```diff
@@ -110,7 +110,7 @@ export default {
               <div>
                 <div :style="{'align-items':'center'}" class="row mb-20">
                   <div class="col span-6">
-                    <LabeledInput v-model="container.name" :mode="mode" :label="t('workload.container.containerName')" />
+                    <LabeledInput v-model="allContainers[0].name" :mode="mode" :label="t('workload.container.containerName')" />
```

Comment:
Should it not be `tab.name`?
Also why always the first one? If there's a valid reason, maybe we could write a comment.